NLET.py

```diff
@@ -64,13 +66,14 @@ def GOButtonCallback(self, widget, data = None):
         #elif self.ECbuttonOTHERCAP.get_active() == True:
         #    OTHERCLD.Pop_UP_OTHERCLD_Window()
 
-        # ----  Some weirdo ANOMALOUS EVENT
-#        elif self.ECbuttonANOMALOUSEVENT.get_active() == True:
-#            self.ProcessNonCTI(self.Event_Tracking_output_filespec, "Anomaly", date, '---', description, "Final")
+        # ----  Show me: FEP_ON_FEP_OFF EVENT
+        elif self.ECbuttonFEP_ON_FEP_OFF_EVENT.get_active() == True:
```

Comment:
Minor nit, but couldn't this just be `elif self.ECbuttonFEP_ON_FEP_OFF_EVENT.get_active():`?